src/ConfigReader/ConfigResult.cs

```diff
@@ -11,4 +11,5 @@ public class ConfigResult
     public bool WriteHeader;
     public string? Header;
     public List<string> TocExcludes = new List<string>();
+    public bool TreatMissingSnippetsAsErrors { get; set; }
```

Comment:
the others are fields. prob keep this consistent?

Reply:
yeah, oversight on my part.